### DISCUSSION TITLE
Opentoonz interface freezes when using Skeleton and Animate Tool under Linux

### DISCUSSION THREAD

**Discussion by ave4** [opentoonz/opentoonz#6756] [Troubleshooting]
When using the Skeleton tool in any mode under Linux, there is a strong delay and 100% loading of a single processor core. Even a simple mouse movement in the scene area causes a 100% CPU load.

[bug1-2019-06-22_13.47.39.mkv.zip](https://github.com/opentoonz/opentoonz/files/3316676/bug1-2019-06-22_13.47.39.mkv.zip)

### System Information
* **OpenToonz Version**: latest nightly release
* **Operating System**: Arch Linux x64
* **CPU**: Intel i3-3220
* **Memory**: 6 Gb
* **Graphics Card**: Intel 
* **Graphics Tablet**:

**Comment by ave4** [comment]
I noticed that when using the All mode in Animate Tool, everything also freezes for a few seconds. Moreover, the OpenToonz interface freezes completely.

**Comment by ave4** [comment]
Video: 
[ot2.mkv.zip](https://github.com/opentoonz/opentoonz/files/3514250/ot2.mkv.zip)

**Comment by ave4** [comment]
There seems to be a problem with these buttons.
![Снимок экрана от 2019-08-18 10-40-53](https://user-images.githubusercontent.com/35303790/63240128-c2f88e00-c267-11e9-9427-7e5249542ebe.png)

**Comment by RodneyBaker** [comment]
Possibly also related to #2046

**Comment by ave4** [comment]
As I identified the problem is with the **intel hd graphics** driver, since everything is fine on radeon.

**Comment by ybabel** [comment]
Same here (v1.4 - ubuntu 20.04 - Intel Corporation UHD Graphics 605) skeleton tool is very slow
I have checked on another configuration, same 1.4 Ubuntu 20.04 but different graphic card : no freeze.
I have checked with v1.4 4-dec-2020 build : still freezing on Intel card, but not on the other one.

**Comment by QbieShay** [comment]
Hey there. I have a radeon rx 590 and Opentoonz freezes for a few seconds every time i try to interact with a skeleton (not plastic skeleton). I'm running Linux as well.

**Comment by ave4** [comment]
It seems the new Mesa Crocus driver fixes this problem. Use `MESA_LOADER_DRIVER_OVERRIDE=crocus opentoonz`

**Comment by andeon** [comment]
@RodneyBaker In some cases, it may be necessary to run OpenToonz by setting an environment variable, such as `MESA_LOADER_DRIVER_OVERRIDE`. This is just part of life for Linux users, who often need to work around such situations due to the many possible configurations. 

The issue could be closed, but it would be good to have a place for troubleshooting tips for easy reference.

Tags for reference/search:
LinuxIssue, Troubleshooting

**Comment by RodneyBaker** [comment]
> it would be good to have a place for troubleshooting tips for easy reference.

@shun-iwasawa is working on something that should help there.
Stayed tuned for news on that.

Added 'troubleshooting' label to this report.
